src/main/resources/db/migration/V011__Add_index_for_checking_if_envelope_exists.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX envelopes_file_name_container_idx
+ON envelopes (file_name, container);
```

Comment:
How about reverting the order of the two columns in the index? This way we could use the index when searching for envelopes for the given container.

Reply:
good point